grid/social-media/posts-comments/profile.supr

```diff
@@ -0,0 +1,130 @@
+name = "social-media/posts-comments"
+version = "1.0.0"
+
+"""
+Get comments on a social media post
+"""
+usecase GetPostComments safe {
+  input {
+    """
+    ID
+    Identifier of the post.
+    """
+    id! string!
+    """
+    Page
+    Pagination cursor (obtained from `previousPage` or `nextPage`).
+    """
+    page string
+  }
+
+  result {
+    """
+    Previous page
+    Previous page cursor.
+    """
+    previousPage string
+    """
+    Next page
+    Next page cursor.
+    """
+    nextPage string
+    """
+    Comments
+    List of comments under the post.
+    """
+    comments [Comment]
+  }
+  
+  error {
+    """
+    Title
+    A short, human-readable summary of the problem type.
+    """
+    title!
+
+    """
+    A human-readable explanation specific to this occurrence of the problem.
+    """
+    detail
+  }
+
+  example success {
+      input {
+          id = "be7cdc05-4264-4d47-bc1f-af704d6abf32"
+      }
+
+      result {
+          comments = [
+            {
+                id = "123"
+                created = {
+                    by = "@alberteinstein"
+                    at = 1652271236
+                }
+                text = "Hello, World!"
+            },
+            {
+                id = "124"
+                created = {
+                    by = "@isaacnewton"
+                    at = 1652272236
+                }
+                lastModified = {
+                    by = "@moderator"
+                    at = 1652272256
+                }
+                text = "(deleted.)"
+            }
+        ]
+      }
+  }
+}
+
+model Comment {
+    """
+    ID
+    Identifier of the comment.
+    """
+    id! string!
+
+    """
+    Created
+    Information about the creation of a comment (author, time).
+    """
+    created {
+        """
+        Created by
+        Identification of the author of the comment.
+        """
+        by string
+        """
+        Created at
+        UNIX timestamp (in seconds) of the comment creation.
+        """
+        at number
```

Comment:
I don't like use of UNIX timestamp here, I'd prefer ISO date time string (if I remember correctly, other networks use date/time strings, esp. Facebook & Instagram). If LI provides only a timestamp, let's return this as undefined from map and add it once #257 or similar is merged (since we have TS conversion functions there)

Reply:
Alright; and yes, LinkedIn only provides a UNIX timestamp.

TBF I'd prefer a proper `DateTime` type so that this could be avoided altogether :P Since as it is, there's really no (overwhelmingly) good or obvious reason to go either way (timestamp or ISO string...)
Maybe this should go into a general `CONVENTIONS` file somewhere?